src/UI/Internal/ToggleableList.elm

```diff
@@ -1,4 +1,4 @@
-module UI.Internal.ToggableList exposing (Config, Cover, view)
+module UI.Internal.ToggleableList exposing (Config, Cover, defaultRow, selectedRow)
```

Comment:
Do we need to expose `defaultRow` and `selectedRow`?

Reply:
We can't import the main list without creating a recursive import. So what leaves is to let this file atomic and keep it internal, as I did here.